# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/jsapi/TransceiverImpl.cpp
# Commit: 2f1e8f5b2fc2
# Full Hash: 2f1e8f5b2fc2f5a28fd2f25019108e56de0fbb44
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Rework GMP PluginID plumbing in VideoCondit. r=bwc
#   
#   This prepares VideoConduit for the decoder/encoder factory and its handling of
#   PluginID.
#   
# ==============================================================================

diff -r 777632b3319a -r 2f1e8f5b2fc2 dom/media/webrtc/jsapi/TransceiverImpl.cpp
--- a/dom/media/webrtc/jsapi/TransceiverImpl.cpp	Fri Feb 05 10:51:01 2021 +0100
+++ b/dom/media/webrtc/jsapi/TransceiverImpl.cpp	Mon Feb 08 17:23:44 2021 +0100
@@ -339,7 +339,7 @@
 }
 
 bool TransceiverImpl::ConduitHasPluginID(uint64_t aPluginID) {
-  return mConduit ? mConduit->CodecPluginID() == aPluginID : false;
+  return mConduit && mConduit->HasCodecPluginID(aPluginID);
 }
 
 bool TransceiverImpl::HasSendTrack(